packages/backend-app-api/src/services/implementations/httpAuth/httpAuthServiceFactory.ts

```diff
@@ -173,12 +173,17 @@ class DefaultHttpAuthService implements HttpAuthService {
 
   async issueUserCookie(
     res: Response,
-    options?: { credentials?: BackstageCredentials },
+    options?: { credentials?: BackstageCredentials; clear?: boolean },
   ): Promise<{ expiresAt: Date }> {
     if (res.headersSent) {
       throw new Error('Failed to issue user cookie, headers were already sent');
     }
 
+    if (options?.clear) {
+      res.clearCookie(BACKSTAGE_AUTH_COOKIE);
+      return { expiresAt: new Date(0) };
```

Comment:
The caller should ignore the return...
Is it possible to specify that the function returns a `Promise<void>` only when the clear option is passed?

Reply:
We are not using clear anymore.